File: java/common-support/src/main/java/net/multi/language/common/marker/message/Channels.java
package net.multi.language.common.marker.message;

public final class Channels {

    // commands
    public static class Commands {
        public static final String CHARGE_ORDER = "charge_order";
        public static final String CREATE_ORDER = "create_order";
    }

    // events
    public static class Events {
        public static final String ORDER_CREATED = "order_created";
        public static final String ORDER_CHARGED = "order_charged";
        public static final String ORDER_NOT_CHARGED = "order_not_charged";
    }

    // queries
    public static class Queries {
        public static final String FIND_ORDER = "find_order";
    }

    // deny instantiation
    private Channels() {}
}

File: python/src/common/marker/message/channels.py
class Channels:
    class Commands:
        CHARGE_ORDER = "charge_order"
        CREATE_ORDER = "create_order"

    class Events:
        ORDER_CREATED = "order_created"
        ORDER_CHARGED = "order_charged"
        ORDER_NOT_CHARGED = "order_not_charged"

    class Queries:
        FIND_ORDER = "find_order"

    def __new__(cls):
        raise TypeError("Channels is not instantiable")
